src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validation.Email;
+
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
     private String password;
```

Comment:
Here we can add annotation @Size with min value

Reply:
we don't have value for the password field in our task specification